tests/test_exercises.py

```diff
@@ -42,23 +42,28 @@ def test_get_all_exercises(self):
         self.assertEquals(payload['exercises'][1]['duration'], '15:00')
         self.assertEquals(payload['exercises'][1]['category'], 'SomaticCategory.MOVEMENT')
 
-    def test_create_exercise(self):
-        response = self.test_app.post(
-            '/api/exercises',
+    def test_find_random_exercise(self):
```

Comment:
AND a test! I love that once we get one thing up and running it is easy to transfer over!